Gemfile.lock

```diff
@@ -37,7 +37,7 @@ GEM
       execjs
       multi_json (>= 1.3)
       rake
-    json (1.8.3)
+    json (2.1.0)
```

Comment:
Dependencies upgrade is already a part of another PR: #1715. 
Could you please revert this change, so Travis CI doesn't break? Thanks!